Author: Thierry Reding <thierry@doppeltgemoppelt.de>
Description: Fix makefile to generate lowercase name for executable.
 Enable debian/rules to pass additional CFLAGS.
Index: billard-gl-1.75/src/Makefile
===================================================================
--- billard-gl-1.75.orig/src/Makefile	2013-06-19 18:29:12.000000000 -0400
+++ billard-gl-1.75/src/Makefile	2013-06-19 18:29:45.000000000 -0400
@@ -7,12 +7,12 @@
 
 CC	=	gcc
 CXX	=	g++
-CFLAGS	=	-pipe -Wall -W -DNO_DEBUG -g
-CXXFLAGS=	-pipe -Wall -W -DNO_DEBUG -g 
+CFLAGS	:=	-pipe -Wall -W -DNO_DEBUG $(CFLAGS)
+CXXFLAGS:=	-pipe -Wall -W -DNO_DEBUG $(CPPFLAGS)
 INCPATH	=	-I/usr/X11R6/include 
 LINK	=	g++
-LFLAGS	=	
-LIBS	=	-L/usr/X11R6/lib -lGL -lGLU -lglut -lXmu -lXext -lX11 -lm -lXi
+LFLAGS	=	$(LDFLAGS)
+LIBS	=	-lGL -lGLU -lglut
 
 ## -lqgl -lGLU
 
@@ -109,7 +109,7 @@
 SRCMOC	=	
 OBJMOC	=	
 DIST	=	
-TARGET	=	BillardGL
+TARGET	=	billard-gl
 INTERFACE_DECL_PATH = .
 
 ####### Implicit rules
